UnitsNet/GeneratedCode/Quantity.g.cs

```diff
@@ -1233,5 +1233,121 @@ public static bool TryParse(IFormatProvider? formatProvider, Type quantityType,
                     return false;
             }
         }
+
+        internal static IEnumerable<Type> GetQuantityTypes()
+        {
```

Comment:
Something tells me we may want to allocate an array for this if it's going to be called multiple times, but that is a minor optimization for later.